video/qt_decoder.cpp

```diff
@@ -926,9 +924,9 @@ void QuickTimeDecoder::VideoTrackHandler::setDither(const byte *palette) {
 			desc->_videoCodec->setDither(Image::Codec::kDitherTypeQT, palette);
 		} else {
 			// Forced dither
-			_forcedDitherPalette = new byte[256 * 3];
-			memcpy(_forcedDitherPalette, palette, 256 * 3);
-			_ditherTable = Image::Codec::createQuickTimeDitherTable(_forcedDitherPalette, 256);
+			_forcedDitherPalette = new Graphics::Palette(256);
```

Comment:
This now leads to mismatched `delete[] _forcedDitherPalette` in the destructor

Reply:
And in other video players too.

Reply:
Nice catch. I'll check more thoroughly for these.

Reply:
I think I've checked all the mismatched deletes.